### DISCUSSION TITLE
Best way to avoid these

### DISCUSSION THREAD

**Discussion by jsbowling42** [blakeblackshear/frigate#5091] [Ask A Question]
ive been playing around with the config of frigate to minimize fasle positives. because of this event:
![image](https://user-images.githubusercontent.com/35424582/212494732-81861ef4-d703-422c-a3bf-5fdf67f25d5a.png)
 ive added a minimum area for people. was wanting to know if this is the best way to handle this. thoughts?

**Comment by NickM-27** [comment]
Yeah I'd definitely just set min_area for person.
